src/Components/GeneralComponents/GeneralWindow.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import GeneralCloseButton from 'Components/GeneralComponents/GeneralCloseButton';
+
+export default function GeneralWindow({ windowPosition, children, handleClose }) {
+  const isWindowRightSided = windowPosition === 'right' || windowPosition === 'Right';
```

Comment:
inside `consts/generalStyles.js`
const windowPositionsOptions = {
right,
left,
top
bottom
}
on `generalWindow.js` :
`const isWindowRightSided = windowPosition === windowPositionsOptions.right`

on `asd.js` (that uses generalWindow):
<GeneralWidnow windowPositon={windowPositionsOptions.right}>

Reply:
maybe make as a function on this component? as this is only relevant to this component.